playbooks/roles/kibana/templates/000-kibana.conf

```diff
@@ -0,0 +1,44 @@
+<VirtualHost *:80>
+    ServerName {{ kibana_server_name }}
+    RewriteEngine On
+    RewriteCond %{HTTPS} !=on
+    RewriteRule ^/?(.*) https://%{HTTP_HOST}:{{ kibana_ssl_port }}/$1 [R,L]
+</VirtualHost>
+
+<VirtualHost *:{{ kibana_ssl_port }}>
+    ServerName {{ kibana_server_name }}
+
+    LogLevel  {{  kibana_log_level|default('info') }}
+    ErrorLog  /var/log/apache2/kibana-error.log
+    CustomLog /var/log/apache2/ssl_access.log combined
+    Options +FollowSymLinks
+
+    SSLEngine on
+    SSLCertificateFile    {{ kibana_ssl_cert|default('/etc/ssl/certs/apache.cert') }}
```

Comment:
We'll need to look at the work done in https://review.openstack.org/171838 and try to make use of the same mechanism so that Kibana is also secure by default.